apps/emqx_connector/src/emqx_connector_mysql.erl

```diff
@@ -143,12 +140,12 @@ maybe_add_password_opt(undefined, Options) ->
 maybe_add_password_opt(Password, Options) ->
     [{password, Password} | Options].
 
-on_stop(InstId, #{poolname := PoolName}) ->
+on_stop(InstId, #{pool_name := PoolName}) ->
     ?SLOG(info, #{
         msg => "stopping_mysql_connector",
         connector => InstId
     }),
-    emqx_plugin_libs_pool:stop_pool(PoolName).
+    emqx_resource_pool:stop(PoolName).
```

Comment:
Note. We can't get rid of the `pool_name` state bit altogether because of implications of #8247, see #10362.

Reply:
Ah. We probably can. But the latter is a good question. I'll address it in #10362.